src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -3572,6 +3572,134 @@ callback (mongoc_client_session_t *session,
 }
 /* End Transactions withTxn API Example 1 */
 
+static void
+_test_sample_versioned_api_example_1 (void)
+{
+   /* Start Versioned API Example 1 */
+   mongoc_client_t *client = NULL;
+   mongoc_server_api_t *server_api = NULL;
+   mongoc_server_api_version_t server_api_version;
+   bson_error_t error;
+
+   /* For a replica set, include the replica set name and a seedlist of the
+    * members in the URI string; e.g.
+    * uri_repl = "mongodb://mongodb0.example.com:27017,mongodb1.example.com:" \
+    *    "27017/?replicaSet=myRepl";
+    * client = test_framework_client_new (uri_repl);
```

Comment:
Though I see this in other examples, it seems wrong to refer to a test function in examples meant for the public docs.

```suggestion
    * client = mongoc_client_new (uri_repl);
```